en/core-api/bank-transfer.md

```diff
@@ -1,69 +1,53 @@
+<ol><li>BCA Virtual Account</li>
+    <li>BNI Virtual Account</li>
+    <li>BRI Virtual Account</li>
+    <li>Mandiri Bill Payment</li>
+    <li>Permata Virtual Account</li></ol>
+For each respective bank, Midtrans creates a virtual account which is allocated to merchant.
+
+?> Please make sure to create your [Midtrans account](/en/midtrans_account/overview), before proceeding with this section.
 <details>
 <summary><b>Sequence Diagram</b></summary>
 <article>
-The overall Bank Tranfer end-to-end payment proccess can be illustrated in following sequence diagram:
 
+The overall Bank Tranfer end-to-end payment proccess is illustrated in following sequence diagram:
 ![bank transfer sequence diagram](./../../asset/image/core_api-sequence_bank_transfer.png)
 </article>
 </details>
 
-Charge API request should be done from Merchant's backend. Server Key (from your account's Dashboard) will be needed to [authenticate the request](https://api-docs.midtrans.com/#http-s-header).
-
-#### Request Details
-Type | Value
---- | ---
-HTTP Method | `POST`
-API endpoint (Sandbox) | `https://api.sandbox.midtrans.com/v2/charge`
-API endpoint (Production) | `https://api.midtrans.com/v2/charge`
-
-#### HTTP Headers
-```
-Accept: application/json
-Content-Type: application/json
-Authorization: Basic AUTH_STRING
-```
-
-**AUTH_STRING**: Base64(`ServerKey + :`)
-
-?> HTTP request by using Basic Authentication method. The username is your Server Key while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by **:** (colon symbol).
-
-## 1. Send Transaction Data to API Charge
-API request should be done from **Merchant’s backend** to acquire `va_number`. There are several components that are required:
-
-Requirement | Description
---- | ---
-Server Key | Explained on [previous section](/en/midtrans-account/overview.md)
-`order_id` | Transaction order ID, defined from your side
-`gross_amount` | Total amount of transaction, defined from your side
-`payment_type` | Set Bank Transfer payment method
-
-Charge API request should be done from Merchant's backend.
-
-### Charge API request
-This is example of basic `/charge` API request in Curl, please implement according to your backend language (you can also check our [available language libraries](/en/technical-reference/library-plugin.md)). The example below shows a sample codes to obtain transaction token:
+## Sandbox Environment
+
+The steps given below use [Midtrans *Sandbox* environment](https://account.midtrans.com/) to test the integration process. Please make sure that you use the *Server Key* and *Client Key* for the *Sandbox* environment. For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys).
+##  Steps for Integration
+To integrate with Bank Transfer payment method, follow the steps given below.
+#### 1. Sending Transaction Data to API Charge
```

Comment:
The page structure is messed up, please be mindful with the header level. Each header level `#`, `##`, `###` is meaningful, do not change the level if you do not understand it.
<img width="265" alt="Screen Shot 2020-11-03 at 17 04 28" src="https://user-images.githubusercontent.com/13027142/97972266-d6edc880-1df6-11eb-838a-361846855f5f.png">